docs/configuration.md

```diff
@@ -931,6 +936,89 @@ Microsoft Teams notifications are sent via the [Incoming Webhooks](https://learn
 [ http_config: <http_config> | default = global.http_config ]
 ```
 
+### `<jira_config>`
+
+JIRA notifications are sent via [JIRA Rest API v2](https://developer.atlassian.com/cloud/jira/platform/rest/v2/intro/)
+or [JIRA REST API v3](https://developer.atlassian.com/cloud/jira/platform/rest/v3/intro/#version).
+
+Note: This integration is only tested against an Jira Cloud instance. Jira Data Center (on premise instance) can work, but it's not guaranteed.
+
+Both APIs have the same feature set. The difference is that V2 supports [Wiki Markup](https://jira.atlassian.com/secure/WikiRendererHelpAction.jspa?section=all)
+for the issue description and V3 supports [Atlassian Document Format (ADF)](https://developer.atlassian.com/cloud/jira/platform/apis/document/structure/).
+The default `jira.default.description` template only works with V2.
+
+```yaml
+# Whether to notify about resolved alerts.
+[ send_resolved: <boolean> | default = true ]
+
+# The URL to send API requests to. The full API path must be included.
+# Example: https://company.atlassian.net/rest/api/2/
+[ api_url: <string> | default = global.jira_api_url ]
+
+# The project key where issues are created.
+project: <string>
+
+# Issue summary template.
+[ summary: <tmpl_string> | default = '{{ template "jira.default.summary" . }}' ]
+
+# Issue description template.
+[ description: <tmpl_string> | default = '{{ template "jira.default.description" . }}' ]
+
+# Add labels to issues
+static_labels: 
+  [ - <string> ... ]
+  
+# Add specific group labels to issue
+group_labels: 
+  [ - <string> ... ]
+
+# Priority of issue
+[ priority: <tmpl_string> ]
+
+# Type of issue, e.g. Bug
+[ issue_type: <string> ]
+
+# Name of the workflow transition to resolve an issue. The target status must have the category "done"
+[ resolve_transition: <string> ]
+
+# Name of the workflow transition to reopen an issue. The target status should not have the category "done"
+[ reopen_transition: <string> ]
+
+# If reopen_transition is defined, ignore issues with that resolution
+[ wont_fix_resolution: <string> ]
+
+# If reopen_transition is defined, reopen issue not older than ... Note: The resolutiondate field is used to determine the age of the issue.
+[ reopen_duration: <duration> ]
+
+# Other issue and custom fields
+fields:
```

Comment:
From https://developer.atlassian.com/server/jira/platform/jira-rest-api-examples/#setting-custom-field-data-for-other-field-types

> A custom field is actually referenced by customfield\_ + the field ID, rather than just the field ID.

I'd rather keep a dedicated "custom fields" entry and have users providing the field IDs:

````
custom_fields:
  "10000": "foo"
  "10001": ["bar", "fred"]
``

Reply:
In conclusion, 2 separate configs for fields?

```
fields:
  component: { name: "Monitoring" }
custom_fields:
  "10000": "foo"
  "10001": ["bar", "fred"]
```

I don't feel thats worth to have 2 here. I renamed it from `custom_fields` to `fields` because it would support jira native fields, too.

I personally would prefer a 1:1 mapping between the field configuration and Jira REST API

Reply:
I agree, mapping with the Jira API makes sense. If they use a string convention rather than a separate API, we should follow the same. Perhaps document/link to this convention here.

Reply:
After more thinking, I agree with both of you.